src/main/java/Krot.java

```diff
@@ -0,0 +1,146 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.util.List;
+
+public class Krot {
+    static String botName = "Krot"; // Static variable for the bot's name
+    static List<Task> tasks = new ArrayList<>(); // Static variable list of all the tasks
+    static boolean hasEnded = false; // Static variable to end the chatbot
+
+    public static String readInput() {
+        // Reads the input and returns the input as a string
+        System.out.println("You:");
+        Scanner in = new Scanner(System.in);
+        return in.nextLine();
+    }
+
+    public static void printName() {
+        // Prints the name of bot when replying
+        System.out.println(botName + ":");
+    }
+
+    public static void greeting() {
+        // Prints out the standard greeting
+        System.out.println("Hello! I'm " + botName);
+        System.out.println("What can I do for you?");
+        printSeparator();
+    }
+
+    public static void printSeparator() {
+        // Print line separators
+        System.out.println("-");
+    }
+
+    public static void endChat() {
+        // Prints closing message
+        printSeparator();
+        printName();
+        System.out.println("Bye. Hope to see you again soon!");
+        hasEnded = true;
+    }
+
+    public static void createTask(String key, String line) {
+        // Prints echo
+        printSeparator();
+        printName();
+        String title;
+        Task task;
+        System.out.println("More tasks to do! I've added:");
+        if (key.equalsIgnoreCase("todo")) {
+            title = line.split(" ", 2)[1];
+            task = new Todo(title, "T");
+            tasks.add(task);
+        } else if (key.equalsIgnoreCase("deadline")) {
+            title = line.split("/")[0].split(" ", 2)[1].strip();
+            String end = " (by:" + line.split("/by", 2)[1] + ")";
+            task = new Deadline(title, end, "D");
+            tasks.add(task);
+        } else {
+            title = line.split("/")[0].split(" ", 2)[1].strip();
+            String[] datesArr = line.split("/");
+            String start = "from: " + datesArr[1].split(" ", 2)[1];
+            String end = "to: " + datesArr[2].split(" ", 2)[1];
+            task = new Event(title, "E", " (" + start, end + ")");
+            tasks.add(task);
```

Comment:
Avoid the usage of magic literals by declaring them as static final variables instead. https://nus-cs2113-ay2324s2.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-magic-numbers